.gitignore

```
@@ -31,3 +31,4 @@ test-report.xml
 .vscode/launch.json
 .vscode/settings.json
 src/jsconfig.json
+.idea/*
```

This doesn't need a `/*` at the end:
```suggestion
.idea
```